modules/client-configuration/pages/supported-features.adoc

```diff
@@ -119,6 +119,11 @@ ifeval::[{suma-content} == true]
 | {x86_64}, {aarch64}
 | {check}
 
+// Raspbian OS 12 not available at Uyuni for now
```

Comment:
Not sure if this will work, did you check it?

Usually I just not add this to SUSE Manager.